packages/types/package.json

```diff
@@ -38,6 +38,7 @@
 		"@types/react": "*",
 		"resolve-tspaths": "*",
 		"tsc-watch": "*",
-		"typescript": "*"
+		"typescript": "*",
+		"@apollo/client": "^3.11.4"
```

Comment:
Should `@apollo/client` be a devDep or an _optional_ peer dep?

Reply:
This is a `devDep` only required when building.